openedx/core/djangoapps/content/search/api.py

```diff
@@ -732,6 +774,30 @@ def update_library_components_collections(
         _update_index_docs(docs)
 
 
+def upsert_library_container_index_doc(container_key: LibraryContainerLocator) -> None:
+    """
+    Creates, updates, or deletes the document for the given Library Container in the search index.
+
+    TODO: add support for indexing a container's components, like upsert_library_collection_index_doc does.
```

Comment:
This would be part of another task?

Reply:
Not sure if we need this right now as [search is not planned yet for units](https://github.com/openedx/frontend-app-authoring/issues/1609). If we add containers to each component index document similar to collections, we'll need to update this function to reindex each child component.